python/lsst/ts/watcher/watcher_csc.py

```diff
@@ -113,19 +113,40 @@ def alarm_callback(self, alarm):
             force_output=True,
         )
 
-    def report_summary_state(self):
-        super().report_summary_state()
+    async def handle_summary_state(self):
         self._enable_or_disable_model()
 
     def do_acknowledge(self, data):
         self.assert_enabled("acknowledge")
         self.model.acknowledge_alarm(name=data.name, severity=data.severity, user=data.acknowledgedBy)
 
     def do_mute(self, data):
+        """Mute one or more alarms.
+        """
         self.assert_enabled("mute")
         self.model.mute_alarm(name=data.name, duration=data.duration,
                               severity=data.severity, user=data.mutedBy)
 
+    async def do_showAlarms(self, data):
+        """Show all alarms.
+        """
+        # First get a list of active (not nominal) alarms.
+        # We can iterate over this list, reporting alarm events and yielding
+        # yielding the event loop, without the list changing.
+        # Note: it is not a problem if alarms change state while this command
+        # is running because those state changes will simply trigger
+        # additional alarm events.
+        active_alarms = [rule.alarm for rule in self.model.rules.values() if not rule.alarm.nominal]
+        print(f"showAlarms found these active alarms: {active_alarms}")
```

Comment:
Any reason for using `print` here and not `log`?

Reply:
Good catch. In fact it was purely for diagnostics while I worked on the code. I removed it.

I also rewrote the code comment. I hope it is clearer now.